extensions/amp-base-carousel/1.0/base-carousel.js

```diff
@@ -49,21 +47,32 @@ export function BaseCarousel({
     carouselContext.setCurrentSlide ?? setCurrentSlideState;
```

Comment:
Not from this PR, but I just checked and it doesn't look like `CarouselContext` has `currentSlide` in its definition. We should either add it or remove the assignment from it

https://github.com/ampproject/amphtml/blob/385ff937ee5defac80ce4a4266d6bd1efac51175/extensions/amp-base-carousel/1.0/carousel-context.js#L18-L23

Reply:
I'm seeing it in the definition. It's in the `type.js` file. Might need to be adjusted to clearly document `|undefined` part though. I'll add that.

Reply:
Done.